configure.ac

```diff
@@ -1954,6 +1967,11 @@ fi
 case $add_players in
   *omxplayer*)
       XB_ADD_PLAYER([OMXPLAYER], [omxplayer])
+      AC_DEFINE([TARGET_RASPBERRY_PI],[1],["Define to 1 for RASPBERRY_PI"])
```

Comment:
and where is the problem with that? RPI only has 1 CPU, RPI2 has another. There is no sense in trying to build with TARGET_RASPBERRY_PI set, but CPU e.g. x86_64